query-connector/e2e/customize_query.spec.ts

```diff
@@ -0,0 +1,193 @@
+// @ts-check
+
+import { test, expect } from "@playwright/test";
+import { TEST_URL } from "../playwright-setup";
+import { PAGE_TITLES } from "@/app/query/stepIndicator/StepIndicator";
+
+import { TEST_PATIENT, TEST_PATIENT_NAME } from "./constants";
+
+test.describe("querying with the Query Connector", () => {
+  // Start every test by navigating to the customize query workflow
+  test.beforeEach(async ({ page }) => {
+    await page.goto(TEST_URL);
+    await page.getByRole("button", { name: "Go to the demo" }).click();
+
+    // Check that the info alert is visible and contains the correct text
+    const alert = page.locator(".custom-alert");
+    await expect(alert).toBeVisible();
+    await expect(alert).toHaveText(
+      "This site is for demo purposes only. Please do not enter PII on this website.",
+    );
+    await expect(
+      page.getByRole("heading", { name: PAGE_TITLES["search"], exact: true }),
+    ).toBeVisible();
+
+    await page.getByRole("button", { name: "Fill fields" }).click();
+    await page.getByLabel("First Name").fill(TEST_PATIENT.FirstName);
+    await page.getByLabel("Last Name").fill(TEST_PATIENT.LastName);
+    await page.getByLabel("Date of Birth").fill(TEST_PATIENT.DOB);
+    await page.getByLabel("Medical Record Number").fill(TEST_PATIENT.MRN);
+    await page.getByLabel("Phone Number").fill(TEST_PATIENT.Phone);
```

Comment:
Do we have to fill these fields if you already clicked "Fill Fields" on line 25?

Reply:
Good question 🤔 . I just copied this setup stuff from the initial part of the regular query_spec tests, but I'd imagine not actually. As long as that test patient remains the same we shouldn't need them.